src/Avalonia.Controls/DataValidationErrors.cs

```diff
@@ -22,8 +22,8 @@ public class DataValidationErrors : ContentControl
         /// <summary>
         /// Defines the DataValidationErrors.Errors attached property.
         /// </summary>
-        public static readonly AttachedProperty<IEnumerable<Exception>> ErrorsProperty =
-            AvaloniaProperty.RegisterAttached<DataValidationErrors, Control, IEnumerable<Exception>>("Errors");
+        public static readonly AttachedProperty<IEnumerable<object>> ErrorsProperty =
```

Comment:
Assuming `DataValidationException` inherits from a simple `Exception`, why use `object` here and not the base type?